docs/guides/connecting_to_your_data/components/where_to_run_code.md

```diff
@@ -0,0 +1,38 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Get an environment to run the code in this guide. Please choose an option below.
+
+<Tabs
+  groupId="yaml-or-python"
+  defaultValue='cli'
+  values={[
+  {label: 'CLI + filesystem', value:'cli'},
+  {label: 'No CLI + filesystem', value:'yaml'},
+  {label: 'No CLI + no filesystem', value:'python'},
+  ]}>
+  <TabItem value="cli">
+
+If you use the Great Expectations CLI, run this command to automatically generate a pre-configured jupyter notebook. Then you can follow along in the YAML-based workflow below:
```

Comment:
I think this will address any potential "mismatch" we have between the tabs here and the tabs below. Thanks!